validator/nodejs/postinstall-windows.js

```diff
@@ -23,7 +23,7 @@ var fs = require('fs');
 if (process.env.OS !== 'Windows_NT') {
   console./*OK*/ error(
       'postinstall-windows.js: This script is for Windows only.');
-  process.exit(1);
+  process.exit(0);
```

Comment:
I don't understand why we'd want to exit with success after deciding that we have an error just one line above.